### PR TITLE
Make Windows binaries natively extractable

### PR DESCRIPTION
**Pull request by mirh**
If you want to replicate the thing in the future, the secret is opening cmd inside the folder you want to compress, then running `%path-to-bat%\script.bat * testcab`

### CODE REVIEW

**Comment by RippeR37** [comment]
I'm sorry, but while `.zip` files are pretty obvious how they should be handled, `.cab` files are not and most people doesn't know that it's another archive format. While using `7zip` is not best here, I'm not sure that this would be clearer to users. Also note that Windows have native support for ZIP format, though it's hard to use it from console.

In the future I may change this script to use PowerShell which is more powerful tool, but for now it's good enough. Example: https://stackoverflow.com/questions/27768303/how-to-unzip-a-file-in-powershell.

So to sum up - I'm not particularly fond of using such archive format.

**Comment by mirh** [comment]
I'm not sure what's the problem. 
From windows side, cab is as straightforward as zip. Be it we are talking about opening it in Explorer, 7-zip (which nobody has in _path_, by the way), or whatnot. 
When talking about console instead, it's a ton more manageable.
